internal/http/client.go

```diff
@@ -37,7 +37,7 @@ func (c *httpClient) Get(path string) ([]byte, error) {
 	buff, _ := ioutil.ReadAll(resp.Body)
 
 	if resp.StatusCode != http.StatusOK {
-		return nil, errors.New(resp.Status)
+		return nil, fmt.Errorf("httpClient buffer: '%s', Status %s", string(buff), resp.Status)
```

Comment:
Maybe (also applies to other line):
```suggestion
		return nil, fmt.Errorf("httpClient buffer: %q, Status %s", string(buff), resp.Status)
```